common/courses/notifications.ts

```diff
@@ -148,26 +241,13 @@ export async function sendPupilCoursePromotion(subcourse: Prisma.subcourse, coun
         return !courseSubject || isPupilsSubject;
     });
 
-    const shuffledFilteredPupils = shuffleArray(filteredPupils);
-
-    // get random subset of filtered pupils
-    const seatsLeft = await getCourseFreePlaces(subcourse);
-    const randomPupilSampleSize = Math.min((seatsLeft / PREDICTED_PUPIL_RESPONSE_RATE) * 100, MAX_PROMOTIONS);
-    const randomFilteredPupilSample = shuffledFilteredPupils.slice(0, randomPupilSampleSize);
```

Comment:
Why is this removed? Taking all pupils potentially notifies thousands of users, we had problems with this in the past